parsl/providers/torque/torque.py

```diff
@@ -120,7 +121,12 @@ def _status(self):
             parts = line.split()
             if not parts or parts[0].upper().startswith('JOB') or parts[0].startswith('---'):
                 continue
-            job_id = parts[0]
+            job_id = parts[0]  # likely truncated
+            for long_job_id in job_ids:
+                if long_job_id.startswith(job_id):
+                    logger.debug(f'coersed jobid {job_id} -> {long_job_id}')
```

Comment:
f-strings are not valid in python 3.5 (which Parsl plans to support for the next version or two - see https://github.com/Parsl/parsl/issues/1553). This makes this code fail the test suite (see the big red X next to your commit).

Reply:
Oh okay, that’s an easy fix. Actually, we don’t necessarily even need to have that logging statement. I was just using during tests.